src/main/java/hu/akarnokd/rxjava/interop/RxJavaInterop.java

```diff
@@ -16,6 +16,8 @@
 
 package hu.akarnokd.rxjava.interop;
 
+import io.reactivex.Scheduler;
```

Comment:
Please don't use imports of Rx stuff in this file but use the fully qualified name instead, like the other methods.